src/de/dkfz/b080/co/aceseq/ACESeqMethods.groovy

```diff
@@ -182,13 +195,20 @@ public final class ACESeqMethods {
         return (TextFile) GenericMethod.callGenericTool(ACEseqConstants.TOOL_ESTIMATE_HRD_SCORE, genderFile, cnvParameterFile);
     }
 
-    public static IndexedFileObjects runParallel( String toolID, IndexedFileObjects fileGroup, BaseFile otherFile, String indexParameterName) {
+    static IndexedFileObjects runParallel( String toolID, IndexedFileObjects fileGroup, BaseFile otherFile, String indexParameterName, LinkedHashMap<String, String> parameters = [:]) {
```

Comment:
I am not sure whether this method exists in COWorkflowsBasePlugin. Maybe we could just utilize that?

Reply:
Yes it is there, but is different. I noticed this and checked.